Sofascore2025/EventView.swift

```diff
@@ -0,0 +1,290 @@
+import UIKit
+import SnapKit
+import SofaAcademic
+
+
+class EventView: BaseView {
+	private var _match : Event
+	private var homeTeamImage = UIImageView()
+	private var awayTeamImage = UIImageView()
+	private var homeTeamName = UILabel()
+	private var awayTeamName = UILabel()
+	private var homeTeamScore = UILabel()
+	private var awayTeamScore = UILabel()
+	private var startTime = UILabel()
+	private var clock = UILabel()
+	private var splitLabel = UIImageView()
+	
+	init(match: Event) {
+		self._match = match
+		super.init()
+	}
+	override func addViews() {
+		super.addViews()
+		addSubview(clock)
+		addSubview(startTime)
+		addSubview(splitLabel)
+		addSubview(homeTeamImage)
+		addSubview(awayTeamImage)
+		addSubview(homeTeamName)
+		addSubview(awayTeamName)
+		addSubview(homeTeamScore)
+		addSubview(awayTeamScore)
+	}
+
+	override func styleViews() {
+		super.styleViews()
+		backgroundColor = .white
+		
+		let formatter = DateFormatter()
+		formatter.dateFormat = "HH:mm"
+
+		startTime.text = formatter.string(from : Date(timeIntervalSince1970: TimeInterval(_match.startTimestamp)))
+		
+		homeTeamName.text = _match.homeTeam.name
+		awayTeamName.text =  _match.awayTeam.name
+		
+		if let homeScore = _match.homeScore {
+			homeTeamScore.text = String(homeScore)
+		}
+		if let awayScore = _match.awayScore {
+			awayTeamScore.text = String(awayScore)
+		}
+
+		
+		if let urlHome = _match.homeTeam.logoUrl, let url = URL(string: urlHome) {
+			loadImage(from: url, isHome: true)
+		}
+		if let urlAway = _match.awayTeam.logoUrl, let url = URL(string: urlAway) {
+			loadImage(from: url, isHome: false)
+		}
+		
+		
+//		clock styling -> only font is constant
+		clock.font = UIFont(name: "Roboto-Regular", size: 12)
```

Comment:
takoder mozes napravit neku apstrakciju i onda imat tamo konstante npr ovo nazoves micro i tu bi ti bilo ovak
```suggestion
		clock.font = .micro
```